discovery-provider/es-indexer/src/types/blocknumber_checkpoint.ts

```diff
@@ -0,0 +1,9 @@
+export type BlocknumberCheckpoint = {
+  users: number
+  tracks: number
+  playlists: number
+  follows: number
+  reposts: number
+  saves: number
+  plays: any
```

Comment:
plays could be number too right (in a perfect world)? it's just slot number?
but as written, I don't think the indexer will catch up on plays right? (I think that's probably fine).

Not blocking, but more macro though, I wonder if the blocknumber is the correct key here. We could also just do track_id, playlist_id, etc. which would be monotonic. OR in the solana world, we could use/change this to slot number (which would also be a number). There may be some subtlety to iron out there. cc @jowlee.

Reply:
Using `track_id` is not enough because need to capture updates too... want to know everything that changed since last run, so something like: `blocknumber` `slotnumber` or `updated_at`

Reply:
Yep right. sg.

Reply:
But yeah... plays could use `slotnumber` no problem... also we're not ETLing plays atm so I could just delete references to plays checkpoint throughout.